### ISSUE TITLE
Poor visibility of unread messages on dark themes

### ISSUE THREAD

**Issue by ksz16** [FossifyOrg/Messages#177]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Unread messages are poorly distinguished from read messages if you use a dark theme (on light themes it looks fine).

### Why do you want this feature?

This would make it easier to use the application.

### Additional information

In the latest Fossify Phone update the visibility of missed, received, outgoing calls in Call History tab has been brilliantly improved by adjusting the proper contrast ratio. 
Would it be possible to adjust this ratio also for unread messages in Fossify Messages? Alternatively, a slight change of colors and bold fonts or a change of background color could help.

**Comment by BadIDMan** [comment]
I just started using Fossify Messages application and indeed this would be nice if that issue is fixed. As I see it's open already almost 1,5 year so don't know if it is still on the ToDo list, though...

**Comment by naveensingh** [comment]
Added badges:

<img alt="ignoreImageMinify" src="https://github.com/user-attachments/assets/82a47ff1-8fc4-4b6e-ab89-9d4f9dc00d9d" width=300 />

**Comment by antodc** [comment]
@naveensingh The changes you made today to improve the readability of the message list are fantastic! I cannot wait to use the new version. Your rabbit should be proud of you, at least as much as we are. Thank you!

**Comment by naveensingh** [comment]
[It's up now :tada:](https://github.com/FossifyOrg/Messages/releases/tag/1.5.0)

**Comment by antodc** [comment]
> [It's up now 🎉](https://github.com/FossifyOrg/Messages/releases/tag/1.5.0)

Just installed, and wow!! So many improvements! By far, the best Messages release ever.

**Comment by BadIDMan** [comment]
I don't like this changes.
While there is the same layout of 10 messages on the screen ("condensation" of messages not possible) with small Font size I don't see the added value to move timestamp to the right.
To me it still looks much better to see 3 lines per each message with timestamp in the third line. And of course the best would be to display FULL date and time for each message, not only a time.

The best solution would be to give us through the settings the choice between:
A) 3 lines per message with color background for unread message
B) 2 lines per message with badges information about unread messages

<img alt="ignoreImageMinify" src="https://github.com/user-attachments/assets/db4a5cac-8cb8-4c10-a296-770f3d9b7d48" width=367 />

**Comment by naveensingh** [comment]
> I don't see the added value to move timestamp to the right.

Moving the timestamp to the top-right keeps the list item height small when using larger font sizes, and it matches other messaging apps.

Changing the font size to small doesn't affect the list density because list items have a [minimum height now](https://m3.material.io/components/lists/specs#c3c3cefe-010d-493e-b719-00f708e3f8b3). Note that "font size" is not the same as "display size" (density), which is why **Font size** and **Display size** are two separate options in system settings. 

> the best would be to display FULL date and time for each message, not only a time.

It displays the time for today and only the date for older conversations.

**Comment by BadIDMan** [comment]
"and it matches other messaging apps" - OK, I thought and I had a hope FossifyOrg is trying to be better than "other" developers, want to break a routine and gives us more options, more choices and in that way to win competition with "other" people who create "the same size" of applications with the same design.
A little bit disappointment on FossifyOrg.

"It displays the time for today and only the date for older conversations." - yeah, I know the logic what is displayed for which message received:
- today: time is displayed
- this year except today: only a day of month
- previous year: whole date but without hour and minutes
Honestly: it is a mess. I really would prefer to see the full date with time for every message.

But again: you should try to be unconventional and unique amongst of "other" developers - give us an option what we would like to see and where: as the third line of message or on the right of message in first line.
I'm pretty sure you would get more satisfied users.

**Comment by naveensingh** [comment]
I think this thread is going off-topic (the issue is about unread messages), but I'll clarify a bit:

> matches other messaging apps

Whenever I say that, I usually mean:

 - A radically different design from other popular apps leads to a steep learning curve.
 - When some design pattern is ubiquitous (Google Messages, Signal, WhatsApp, QUIK, etc.), it's likely because it is an effective solution to a common problem (i.e., where do you show the datetime?).
 - Others have already done more research on ideal UI/UX. The big leagues may not have the same motivation, but they can't get what they want with a bad user experience.

>  I thought and I had a hope FossifyOrg is trying to be better than "other" developers, want to break a routine and gives us more options, more choices and in that way to win competition with "other" people who create "the same size" of applications with the same design.

Not exactly. The only goal is to provide high-quality open-source apps with ideal defaults and not too many bells and whistles.
